util/strings.go

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
Can these functions be added as private functions wherever they're used or copy/pasted? They're trivial and adding a `util` package to a util library is too much.

Reply:
Sounds reasonable to me. 👍 

done @ 4fd8d8f9b13ca99da881ea1f1c5f2db463263c0d